src/screens/SplashScreen.tsx

```diff
@@ -0,0 +1,15 @@
+import Lottie from 'lottie-react-native'
+import React, { ReactElement } from 'react'
+
+const SplashScreen = (): ReactElement => {
```

Comment:
so are we not using the react-native-bootsplash anymore? if yes, let's remove all the code related to it. You can see what files are related to it here https://github.com/zoontek/react-native-bootsplash

Reply:
We continue to use that library. works with RNBootSplash -> Splash(Animation) -> Main or Update